.github/scripts/build-image-locally-v3.sh

```diff
@@ -17,12 +17,14 @@
 #
 
 SUBMARINE_VERSION="0.8.0-SNAPSHOT"
```

Comment:
I am consulting on an unrelated matter:
Can we get our version information directly from the `pom.xml` so that we don't need to do a lot of text replacement and version number validation each time we make a change to the version? For example
```shell
awk '/<\/*version\/*>/{gsub(/[[:space:]]*<\/*version\/*>/,"");print $0}' pom.xml | head -n 1
```

cc @pingsutw @xunliu

Reply:
In my opinion, maybe we could create another text file call submarine.version or something else and it would be the source of project version, all the metadata could read from it, include pom.xml, build script, and anything else.

Reply:
@FatalLin 
Created a new issue https://issues.apache.org/jira/browse/SUBMARINE-1328
I will follow up on this issue further.